Customer.cs

```diff
@@ -52,7 +52,7 @@ public string paymentCalculation(string loginDate, string exitDate)
             TimeSpan result = _exitDate - _loginDate;
 
             int manyDay = Convert.ToInt32(result.TotalDays);
-
+            _price = manyDay * 50;
```

Comment:
buraya sonrasında sabit bir değişken oluşturmayı ve onun üzerinden hesaplama yapmayı düşünüyorum.

Reply:
Bu 50 sanirim gecelik fiyat oluyor. Odalarin farkli fiyatlari olursa ya?

Reply:
Yüksek, orta, düşük şeklinde 3 farklı oda fiyat sınırlaması yapsak ve bu fiyatları database de aynı başlıklar altında tutsak. Mesela herhangi bir zamlanma veya indirim de direkt değiştirilebilir de olsa. nasıl olur sizce ? @zencifil

Reply:
Daha da flexible olmayi dene. Sonucta oda ve fiyati 1-1 bir eslesme. Oda tablosunun icinde tutabilirsin bunu sanki, ne dersin?

Reply:
evet dediğiniz gibi daha mantıklı bir bağlam. sanırım bunu nasıl yapabileceğimi biraz düşünmeli ve araştırmalıyım. :)

Reply:
gün üzerine veya oda üzerine belirlenen  ücreti dışardan alsak ve kalma süresi doğrultusunda toplam ücreti sadece veritabanında tutsak, nasıl olur sizce ? @zencifil

Reply:
kalma suresi ile oda tablosunun nasil bir iliskisi var?

Reply:
aa evet, daha cok customer ile ilişkili bu.
oda için değerleri de odalarla eşleyebilirim, sanırım.